draft-ietf-mls-protocol.md

```diff
@@ -2815,9 +2819,9 @@ Note that the `tree_hash` field is used the same way as in the Welcome message.
 The full tree can be included via the `ratchet_tree` extension
 {{ratchet-tree-extension}}.
 
-The signature MUST verify using the public key taken from the credential in the
-leaf node at position `signer_index`.  The signature covers the following
-structure, comprising all the fields in the PublicGroupState above `signer_index`:
+The signature MUST verify using the public key taken from the credential of the
+`signer`. The signature covers the following structure, comprising all the
+fields in the PublicGroupState above `signer`:
```

Comment:
I actually think the old wording is clearer.  In general, I would still talk about these `Client` values as indices, since you need to look up leaves in the tree to get credentials, signature keys, etc.